samples/bluetooth/fast_pair/locator_tag/configuration/pm_static_nrf54l15dk_nrf54l05_cpuapp_release.yml

```diff
@@ -0,0 +1,55 @@
+mcuboot:
+  address: 0x0
+  region: flash_primary
+  size: 0x5000
```

Comment:
We may need to increase it if we switch to using MCUboot HW crypto at some point

Reply:
Yes, we have a dedicated task for it and we will align the memory partition if we manage to enable HW crypto for MCUboot